### PR TITLE
Bugfixing the close-talks-issues.yaml workflow

### PR DESCRIPTION
**Pull request by sgibson91**
### Summary

This PR is fixing a few bugs in the close-talk-issues.yml workflow

### List of changes proposed in this PR (pull-request)

* Code cleanly exits if there are no issues to close
* Establishing if a newsletter issue exists now works
* Creating a new issue now works

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: .github/workflows/close-talk-issues.yml [hunk 1]
```diff
@@ -133,8 +139,8 @@ jobs:
                   repo=repo,
                   title="Recent talks to promote in the newsletter",
```
Comment (sgibson91) [inline .github/workflows/close-talk-issues.yml:140]:
```suggestion
                  title=":robot: Recent talks to promote in the newsletter",
```

**Review by aldenc — APPROVED** [review]
@sgibson91 do we need to merge this and test again?

**Comment by sgibson91** [comment]
No, with worflows that have a `workflow_dispatch` trigger, you can pick the branch it runs the workflow from (providing a version of the file already exists on the default branch). So the issue I shared in Slack last night was the result of _this_ version of the code. No need to retest :)

**Comment by aldenc** [comment]
perfect!

**Pull request merged**